homeassistant/components/ads/switch.py

```diff
@@ -29,7 +32,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     name = config.get(CONF_NAME)
     ads_var = config.get(CONF_ADS_VAR)
 
-    add_entities([AdsSwitch(ads_hub, name, ads_var)], True)
+    add_entities([AdsSwitch(ads_hub, name, ads_var)])
 
 
 class AdsSwitch(ToggleEntity):
```

Comment:
It would be nice if we refactor and extract the common entity logic into a common base ADS entity class, and avoid the copy paste. We don't need to do that in this PR.

Looks like we can extract:
`async_added_to_hass`, `is_on`, `name`, `unique_id`, `should_poll`, and `available` if we rename some attributes and move out some functions that have platform specific names and logic.